statistics-production/ud.qmd

```diff
@@ -1063,8 +1063,33 @@ Where there is a reasonable need to publish or report statistics for a full aggr
 
 ---
 
+#### Non-GSS standard data collection 
+
+---
+
+##### Black British / Welsh and Asian British Welsh categories
+
+---
+
+From the start of the 2024/25 academic year, four additional ethnicity codes were added to the Common Basic Data Set (CBDS) for use within management information systems and for return to the Department via statutory data collections. These new codes are: 
```

Comment:
Need a blank line before the bullets as it's not formatting right

![image](https://github.com/user-attachments/assets/59091892-5233-491c-a566-c432fa848cc1)

Reply:
Done